src/network/networkctl.c

```diff
@@ -239,13 +242,17 @@ static int list_links(int argc, char *argv[], void *userdata) {
 
         (void) pager_open(arg_pager_flags);
 
-        if (arg_legend)
-                printf("%3s %-16s %-18s %-16s %-10s\n",
-                       "IDX",
-                       "LINK",
-                       "TYPE",
-                       "OPERATIONAL",
-                       "SETUP");
+        table = table_new("IDX", "LINK", "TYPE", "OPERATIONAL", "SETUP");
+        if (!table)
+                return log_oom();
+
+        table_set_header(table, arg_legend);
+        table_set_width(table, 77);
```

Comment:
why explicitly configure the width? i think unless there's a strict reason to we should let the code figure out the width on its own?